src/main/java/seedu/address/model/person/Organization.java

```diff
@@ -0,0 +1,109 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Represents an Organisation in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Organization extends Contact {
+
+    private final Status status;
+    private final Position position;
+    //private final Url url;
+
+    // Data fields
+    //private final Set<Recruiter> tags = new HashSet<>();
+
+    /**
+     * Every field must be present and not null.
+     */
+    public Organization(Name name, Phone phone, Email email, Address address,
+                        Set<Tag> tags, Status status, Position position) {
+        super(name, phone, email, address, tags);
+        requireAllNonNull(status, position);
+        this.status = status;
+        this.position = position;
+    }
+
+    public Status getStatus() {
+        return status;
+    }
+
+    @Override
+    public Boolean isOrganization() {
+        return true;
+    }
+
+    public Position getPosition() {
+        return position;
+    }
+
+    //    public Address getUrl() {
+    //        return url;
+    //    }
+
+    //    public Set<Recruiter> getTags() {
+    //        return Collections.unmodifiableSet(recruiters);
+    //    }
+
+    /**
+     * Returns true if both contacts have the same name.
+     * This defines a weaker notion of equality between two contacts.
+     */
+    public boolean isSameOrganization(Organization otherOrganization) {
+        if (otherOrganization == this) {
+            return true;
+        }
+
+        return otherOrganization != null
+                && otherOrganization.getName().equals(getName());
+    }
+
+    /**
+     * Returns true if both contacts have the same identity and data fields.
+     * This defines a stronger notion of equality between two contacts.
+     */
+    //    @Override
+    //    public boolean equals(Object other) {
+    //        if (other == this) {
+    //            return true;
+    //        }
+    //
+    //        // instanceof handles nulls
+    //        if (!(other instanceof Contact)) {
+    //            return false;
+    //        }
+    //
+    //        Contact otherContact = (Contact) other;
+    //        return name.equals(otherContact.name)
+    //                && phone.equals(otherContact.phone)
+    //                && email.equals(otherContact.email)
+    //                && address.equals(otherContact.address)
+    //                && tags.equals(otherContact.tags);
+    //    }
+
+    //    @Override
+    //    public int hashCode() {
+    //        // use this method for custom fields hashing instead of implementing your own
+    //        return Objects.hash(name, phone, email, address, tags);
+    //    }
```

Comment:
Try not to comment out existing code as a way to mark pending modifications. Instead, if you have a pending change, like needing to add parameters to compare, use `// TODO:` at appropriate areas to specify what's left to do.

No need to change immediately, but remember to uncomment them when you're done.